base/sparse/sparsematrix.jl

```diff
@@ -1332,6 +1332,42 @@ function findnz(S::SparseMatrixCSC{Tv,Ti}) where {Tv,Ti}
     return (I, J, V)
 end
 
+function _sparse_findnext(m::SparseMatrixCSC, i::Int)
+   if i > length(m)
+       return 0
+   end
+   row, col = ind2sub(m, i)
+   lo, hi = m.colptr[col], m.colptr[col+1]
+   n = searchsortedfirst(@view(m.rowval[lo:hi-1]), row)
```

Comment:
You can specify start and stop index to `searchsortedfirst` instead of using a view:
https://github.com/JuliaLang/julia/blob/a945af3aaef528da7813174c17d705b6383f7ea3/base/sort.jl#L160-L172